src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -45,10 +45,11 @@ zookeeper_client_port: 2181
 formplayer_port: 8181
 
 couchdb_use_haproxy: False
+couchdb_use_alb: True
```

Comment:
I think this should default to false. Defaults should be set to what third parties are most likely using / changes should be done so as not to affect them.

Reply:
(it can then be set to true in staging/public.yml)

Reply:
```
Previous: couchdb2_proxy_port: "{{ 35984 if couchdb_use_haproxy else 25984 }}"
Current Change: couchdb2_proxy_port: "{{ 35984 if couchdb_use_haproxy else 80 if couchdb_use_alb else 25984 }}
```
 Danny: Can you make the ALB listen on port 25984 or just the standard 5984? I think that would be a cleaner change
Prem: Yes, we can do that then I think having an additional variable `couchdb_use_alb` is not required, Can I remove it?